pkg/apis/operator/v1alpha1/tektonpipeline_lifecycle.go

```diff
@@ -21,107 +21,145 @@ import (
 	"knative.dev/pkg/apis"
 )
 
-var (
-	_ TektonComponentStatus = (*TektonPipelineStatus)(nil)
```

Comment:
can we add this check back when we refactor the other reconcilers. 
if yes, could you comment this for now instead of removing it. Also mark it as TODO.

Reply:
yes we can add this back later, will add a comment